### PR TITLE
Compile script and update .gitignore

### PR DESCRIPTION
**Pull request by JorikSchellekens**
Add script to compile tests/yul/*.sol
Ignore .warp-deps

### CODE REVIEW

File: scripts/yul/compile_sol_files_in_tests_yul.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+for f in $(ls tests/yul/*.sol); do
```
Comment (murcake) [inline scripts/yul/compile_sol_files_in_tests_yul.sh:1]: Just carping: why not `for f in tests/yul/*.sol;`?

File: .gitignore [hunk 2]
```diff
@@ -5,6 +5,7 @@ vendor/cairo-lang/build
 
 # Secrets
 /.warp-activation-token
+/.warp-deps
```
Comment (murcake) [inline .gitignore:8]: What command generates that?

**Review by murcake — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (JorikSchellekens) [reply .gitignore:8]: Ah, something I'd added to the makefile a long time ago but seems like someone changed the makefile

**Review by murcake — APPROVED** [review]
I'd also add a shebang to the script

**Comment by murcake** [comment]
We should probably remove the ", ignore .warp-deps" from the commit description.

Sorry, for telling this in bits, I didn't notice the last time.

**Pull request merged**